config/01-ce-collector-audit-payload.conf

```diff
@@ -0,0 +1,24 @@
+# Settings for the collector audit payloads function.
+# The easiest way to direct the payload logging information here is to
+# set the environment variable CONDORCE_COLLECTOR_HOST on worker nodes
+# pointing to this condor-ce host.  Then pilot jobs that use condor
```

Comment:
This is automatically done by the CE, so we don't need this comment - should be redundant.